src/assets/i18n/nb.json

```diff
@@ -437,7 +437,8 @@
           "MESSAGE": "Observasjonen inneholder data som ikke kan vises i enkel modus. Vil du slette data vi ikke kan vise?",
           "YES": "Slett data og bytt til enkel",
           "CANCEL": "Avbryt"  
-        }
+        },
+        "IMAGES_MODAL_TITLE": "Bilder av {{postfix}}"
```

Comment:
Husk å legge til engelsk også.

Reply:
For oversetterne er det vanskelig å forstå hva "postFix" er for noe. Variabelnavnet må endres til noe som gir mening for oversetterne.

Reply:
Takk for innspill! Har lagt inn engelsk og endre navn på variabelen "postFix"